docs/products/opensearch/howto/upgrade-to-opensearch.rst

```diff
@@ -48,3 +48,7 @@ Here are some related resources that may help you with your upgrade process
 * :doc:`Upgrade your application client libraries for OpenSearch <upgrade-clients-to-opensearch>`
 * :doc:`Plugins available on for Aiven for OpenSearch <../reference/plugins>`
 * :doc:`OpenSearch Dashboards <../dashboards/index>` as a replacement to Kibana
+
+------
+
+*Elasticsearch is a trademark of Elasticsearch B.V., registered in the U.S. and in other countries.*
```

Comment:
Do we need also to mention Kibana here?

Reply:
Oh, yes, well spotted: https://www.elastic.co/legal/trademarks